functional-test/src/test/groovy/org/rundeck/tests/functional/selenium/jobs/LogViewerOutputSpec.groovy

```diff
@@ -77,4 +78,30 @@ class LogViewerOutputSpec extends SeleniumBase{
             selectedLine
         }
     }
+
+    def "show whale log warning message"() {
+
+        given:
+        def loginPage = page LoginPage
+        def sideBar = page SideBarPage
+        def projectHomePage = page HomePage
+        def jobShowPage = page JobShowPage
+        def executionShowPage = page ExecutionShowPage
+
+        when: "We run the job to have multiple lines in log output"
+        loginPage.go()
+        loginPage.login(TEST_USER, TEST_PASS)
+        projectHomePage.validatePage()
+        projectHomePage.goProjectHome(longOutPutProjectName)
+        sideBar.goTo(NavLinkTypes.JOBS)
+        jobShowPage.goToJob("1f0306cd-e123-44f3-8977-9e52edad8ce7")
+        jobShowPage.getRunJobBtn().click()
+        executionShowPage.validateStatus 'SUCCEEDED'
+        executionShowPage.getLink 'Log Output' click()
+        def showWarningMessage = jobShowPage.waitForElementVisible(By.xpath("//div[contains(@class, 'execution-log__warning')]")).isDisplayed()
```

Comment:
IIUIC, this verifies that the `default` max log size applies. 
However, it does not verify that the max log size is configurable.